### ISSUE TITLE
Does not load file fields when uploading new image from image library

### ISSUE THREAD

**Issue by stpaultim** [backdrop-contrib/file_entity_inline#1]
When adding a new file to an entity from the file browser (image library) it does not populate the added field fields properly and when entity is saved, it overwrites existing data in the custom file field. 

Steps to reproduce:

1) Install and enable the file_entity_inline module
2) Add a "Photo Credit" field to the image file type. 
3) Create NEW POST ONE and add IMAGE ONE (from desktop) with a photo credit = CREDIT ONE.
![image](https://user-images.githubusercontent.com/3144571/188520918-51b3f3e4-7739-4300-bac4-97984135d1c9.png)
![image](https://user-images.githubusercontent.com/3144571/188521000-0d8d5c19-adb9-4ef0-8a81-640565498ccb.png)

4) Go to manage files and verify that IMAGE ONE has a photo credit field that contains CREDIT ONE.
5) Create NEW POST TWO and add IMAGE ONE through the file/image browser
![image](https://user-images.githubusercontent.com/3144571/188520952-7b254d99-5b68-4d9d-b005-ee1b3e2ee76e.png)
![image](https://user-images.githubusercontent.com/3144571/188521033-b1f8aacf-2c65-4a15-b57b-7ac583a136b6.png)
6) BUG - Notice that the photo credit field is empty. 
7) BUG - Save post and view IMAGE ONE through the file manager to see that CREDIT ONE has been removed.
8) Go to file manager and edit the photo credit field for IMAGE ONE to CREDIT TWO and save file.
9) Clear the cache (important)
10) Edit POST ONE and POST TWO and see that IMAGE ONE has a photo credit field that contains CREDIT TWO.
![image](https://user-images.githubusercontent.com/3144571/188521071-88121dd8-1083-41e3-b65e-f5f17a540e20.png)

If while creating a new post you upload a new image and add data to custom file field and save. The data is save correctly. 
If while creating a new post you upload an existing image from file browser the data in the custom field does not load. 
If while creating a new post you upload an existing image from file browser the data does not load and when you save the node, any previous data in the custom file field will be overwritten. 
If you edit the custom field through the manage files interface and then view any nodes with that image, the new data in the custom field is available.

**Comment by bdalomgir** [comment]
I found the problem which is as below -  
1. When generating an inline form it does actually attach form to the entity type file for that specific file object. 
2. But the file object does not have the all inline fields in it rather it has only standard file object without fields. 
Solution : 
If we just load the file object using file_load($fid), It will return whole file object with all fields in it and it fixes the problem. 
I have fixed it adding the following code in line  #74 
`$file = file_load($file->fid);`
And its working for me. I will push the changes soon.
